<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Bioinformatics Tools by mmlantonio</title>
    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/github-dark.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
    <script src="javascripts/main.js"></script>
    <!--[if lt IE 9]>
      <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">

  </head>
  <body>

      <header>
        <h1>Bioinformatics Tools</h1>
        <p>General scripts for dealing with sequencing files and data extraction</p>
      </header>

      <div id="banner">
        <span id="logo"></span>

        <a href="https://github.com/mmlantonio/Bioinformatics-Tools" class="button fork"><strong>View On GitHub</strong></a>
        <div class="downloads">
          <span>Downloads:</span>
          <ul>
            <li><a href="https://github.com/mmlantonio/Bioinformatics-Tools/zipball/master" class="button">ZIP</a></li>
            <li><a href="https://github.com/mmlantonio/Bioinformatics-Tools/tarball/master" class="button">TAR</a></li>
          </ul>
        </div>
      </div><!-- end banner -->

    <div class="wrapper">
      <nav>
        <ul></ul>
      </nav>
      <section>
        <p><b>countTA.pl</b></p>

<p>A perl script that takes a fasta file and counts the number TA sites. 
This is useful for Transposon Sequencing when we want to know the ratio of transposon insertions 
(which happen at TA sites) to TA sites for a given region. For example, the ratio is used in the essentails 
analysis which calculates the p-value for a region given insertion representation</p>

<p><b>getCoordsGBK.py</b></p>

<p>A python script that takes a genbank file and outputs a file with all the gene ids and their genomic start and end coordinates. This script is useful for creating genomic tracks in Gviz (R package).</p>
      </section>
      <footer>
        <p>Project maintained by <a href="https://github.com/mmlantonio">mmlantonio</a></p>
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://twitter.com/michigangraham">mattgraham</a></small></p>
      </footer>
    </div>
    <!--[if !IE]><script>fixScale(document);</script><![endif]-->
    
  </body>
</html>
